core_data_modules/traced_data/io.py

```diff
@@ -308,7 +308,7 @@ def export_traced_data_iterable_to_the_interface(cls, data, export_directory,
             for td in data:
                 row = {
                     "phone": td[phone_key],
-                    "date": datetime.strftime(isoparse(td[date_key]), "%d/%m/%Y"),
+                    "date": datetime.strftime(isoparse(td[date_key]), "%m/%d/%Y"),
```

Comment:
I presume this is what the Interface requires? I'm not fond of this time format in general.

Reply:
Yes. I tired ISO format too, but that also doesn't work.